### ISSUE TITLE
MudSplitter: Multiple contents separated by splitters

### ISSUE THREAD

**Issue by mueller-marcel** [CodeBeamOrg/CodeBeam.MudExtensions#323]
Hello there,

I am interested in the MudSplitter and I would like to use it with multiple contents separated by splitters.
The basic usage of the MudSplitter is with `StartContent` and `EndContent` which alignes some code on the left and the right side of the splitter. How can I achieveto put 3 or more of such contents in a in splitter grid. The following pictures shows what I would like to achieve.

![image](https://github.com/CodeBeamOrg/CodeBeam.MudBlazor.Extensions/assets/48565242/d8a43d26-acf7-44a5-92ba-bad6f2a4f93c)

I found such a solution in Radzen blazor. Instead of a `StartContent`  and an `EndContent` they use `RadzenSplitterPane`, which is a more generic approach to achieve multiple contents separated by splitters.

```
<RadzenSplitter Orientation="Orientation.Vertical">
    <RadzenSplitterPane>
        Pane B2
    </RadzenSplitterPane>
    <RadzenSplitterPane>
        Pane B3
    </RadzenSplitterPane>
    <RadzenSplitterPane>
        Pane B2
    </RadzenSplitterPane>
</RadzenSplitter>
```

I would like to request such a feature for the `MudSplitter`. Maybe this can be realized with the `MudItem` that are used in the `MudGrid` as well. It would look like following

```
<MudSplitter>
    <MudItem>
        Content 1
    </MudItem>
    <MudItem>
        Content 2
    </MudItem>
</MudSplitter
``` 

Best regards
Marcel Müller

**Comment by mckaragoz** [comment]
Hmm, its currently not possible by design. But you can add nested splitters to have multiple areas.

**Comment by mueller-marcel** [comment]
> Hmm, its currently not possible by design. But you can add nested splitters to have multiple areas.

Yes thank you. I used this workaround. Could this be considered a feature in a future release? (tbd)

**Comment by mckaragoz** [comment]
Why not, we can think about whole design of splitter, it currently also doesn't support vertical mode.

**Comment by mueller-marcel** [comment]
> Why not, we can think about whole design of splitter, it currently also doesn't support vertical mode.

Didn't need it yet but vertical splitters also make sense. If you need help with the redesign let me know 👍
